specificationRepositoryConfiguration.json

```diff
@@ -19,6 +19,11 @@
       "integrationRepository": "AzureSDKAutomation/azure-sdk-for-net",
       "mainRepository": "Azure/azure-sdk-for-net"
     },
+    "azure-sdk-for-net-track2": {
```

Comment:
I'm not sure I like calling this track 2 could we instead have it mirror the version of the code generator?

Reply:
IMO you have another set of configuration for .NET track2 here so I would like to make it another "language" for SDK Automation